### PR TITLE
Dplamap / pcdm

### LINKED ISSUE #1: Describe relationship between DPLA MAP and PCDM
Responsible: @no-reply @azaroth42
Approver: @anarchivist?
Consulted: @mjgiarlo @guegueng
Informed: @anarchivist 

Complete when there is a public description of how DPLA MAP and PCDM can be used together.

### PR DESCRIPTION
**Pull request by azaroth42**
closes #1

### CODE REVIEW

File: notes/dpla_map_and_pcdm.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
+## Relationship between DPLA MAP and PCDM
+
+This document notes the ways in which the DPLA Metadata Application Profile and PCDM can be used together.
+
+### Mapping from PCDM to DPLA MAP
+
+#### Collections
+
+One or more `pcdm:Collection`s map to a `dcmitypes:Collection`. As `pcdm:Collection`s may be hierarchical, and DPLA's use of `dcmitypes:Collection` is not hierarchical, information may be accumulated from multiple `pcdm:Collection`s into one.
```
Comment (no-reply) [inline notes/dpla_map_and_pcdm.md]: maybe "Zero or more"?

Comment (azaroth42) [inline notes/dpla_map_and_pcdm.md]: Zero or more collections map? How would you map from no collection to a collection?  ... or ... you can certainly create collections from external information, but that's not the point of this doc?

Comment (anarchivist) [inline notes/dpla_map_and_pcdm.md]: minor typo: `dcmitype:Collection`

File: notes/dpla_map_and_pcdm.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+The resulting collection will be associated with a `dpla:SourceResource` constructed from a `pcdm:Object`.  This is the inverse of the PCDM hierarchy, where Collections have members.
+
+
+#### Objects
+
+PCDM Objects can be used to describe objects (both conceptual and physical), and parts of those objects.  DPLA is mostly concerned with identifiable cultural heritage objects (`dpla:SourceResource`) and the digital views of that object (`dpla:WebResource`).
+
+When mapping from PCDM to DPLA, a `pcdm:Object` that is not `pcdm:hasMember` another `pcdm:Object` will be treated as a `dpla:SourceResource`.  Other resources may also be treated in this way.  These Objects may or may not be part of a `pcdm:Collection`.
+
+Such objects MUST have a stable web URL at which the object is rendered in a human readable way.  This is the object of `dpla:shownAt`.  Such objects SHOULD have a `dpla:WebResource` which is a `preview` of the object, such as a thumbnail.
```
Comment (anarchivist) [inline notes/dpla_map_and_pcdm.md]: `dpla:shownAt` should be `edm:isShownAt`

File: notes/dpla_map_and_pcdm.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+Such objects MUST have a stable web URL at which the object is rendered in a human readable way.  This is the object of `dpla:shownAt`.  Such objects SHOULD have a `dpla:WebResource` which is a `preview` of the object, such as a thumbnail. 
+
+As per Collections, the resulting DPLA resource may conflate information from an entire hierarchy of objects into a single resource. 
+
+#### Files
+
+Some `pcdm:File`s are `dpla:WebResource`s, but not all.  It would be true in at least the following situations:
+
+* The `pcdm:File` is a view of the `dpla:SourceResource` (`dpla:shownAt`)
```
Comment (anarchivist) [inline notes/dpla_map_and_pcdm.md]: `dpla:shownAt` should be `edm:isShownAt`

File: notes/dpla_map_and_pcdm.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+* The `pcdm:File` is a preview / thumbnail of the `dpla:SourceResource` (`dpla:preview`)
```
Comment (anarchivist) [inline notes/dpla_map_and_pcdm.md]: `dpla:preview` should be `edm:preview`

File: notes/dpla_map_and_pcdm.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+### Further Notes
+
+* The reverse mapping is lossy and should not be attempted
```
Comment (anarchivist) [inline notes/dpla_map_and_pcdm.md]: more information here? maybe just 1 or 2 sentences.

**Comment by anarchivist** [comment]
Sidebar: We don't have to do it to get this merged, but this is probably worth at least a short conversation with @aisaac and Valentine, because it would be nice to have a similar document -- or this one -- articulate the relationship between PCDM and EDM.

**Comment by azaroth42** [comment]
:+1: to the suggestion. Is that a HyBox or PCDM concern, thus where to create a new issue to track it?

**Comment by anarchivist** [comment]
@azaroth42 I'm inclined to suggestions there. I think it depends on how PCDM sees alignment with EDM as a priority. :)

**Comment by azaroth42** [comment]
@anarchivist The same applies for IIIF, both PCDM <--> IIIF (/cc @jpstroop) and for IIIF <--> EDM.
Lets make an issue here, and reference it from other repos.

**Comment by azaroth42** [comment]
Made issue #7. Anything outstanding on this PR being committed?

**Comment by anarchivist** [comment]
Any interest or need in squashing the commits?

**Comment by jpstroop** [comment]
Make him squash! :stuck_out_tongue_winking_eye:

**Comment by no-reply** [comment]
:squash:

**Comment by azaroth42** [comment]
Fiiiiine.

**Comment by azaroth42** [comment]
Squashed down to initial plus revision. :+1:?

**Pull request merged**